modules/version-control/version_control_exercise2.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Tool search
+type: exercise
+order: 3
+---
+
+## Exercise: tool search
+
+In breakout groups of 3-4 people, identify 3 commonly used version control systems or tools that you have heard of (or used yourself) (for example: Git). 
```

Comment:
Is this meaning mercurial, svn, fossil etc? Are most students of this course likely to have even heard of one other than git?

Reply:
I'll open an issue to revise both exercises.